buildcontext/excludes.go

```diff
@@ -35,3 +56,14 @@ func readExcludes(dir string) ([]string, error) {
 	}
 	return append(excludes, ImplicitExcludes...), nil
 }
+
+// cleanest way I could imagine to iterate and check if both of them exist - If the file is a bash/executable file then it could return an error other than (os.IsNotExist(err))
```

Comment:
instead of this function, would it be possible to use `fileutil/FileExists` instead? it exists under `util/fileutil/fileutil.go`. I think it would be fine to ignore the errors that are swallowed by `FileExists()`.

Reply:
My apologies - I thought I had checked if there was a function already - I'll apply that